### ISSUE TITLE
Evaluation on the whole test set

### ISSUE THREAD

**Issue by SherlockHua1995** [Neerajj9/CSRNet-keras#4]
hello, I am try to evaluate the trained model on the whole test set (for A and B) to get the MAE of ShanghaiTech part A and ShanghaiTech part B .
Would you plz supply your script? 
Thank you.
